src/screens/TopicPageScreen/elements/BottomSheetFollow.tsx

```diff
@@ -87,6 +92,7 @@ const BottomSheetFollow = forwardRef((props: BottomSheetFollowProps, ref: Ref<RB
     try {
       const isIncognito = type === 'incognito';
       await followTopic(topicName, isIncognito);
+      deleteChannelList(topicId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `deleteChannelList` method is being used here. Similar to the previous comment, consider adding error handling around this operation.
```diff
    try {
       const isIncognito = type === 'incognito';
       await followTopic(topicName, isIncognito);
+      try {
         deleteChannelList(topicId);
+      } catch (error) {
+        console.error('Failed to delete channel list:', error);
+      }
    } catch (error) {
       if (__DEV__) {
         console.log(error);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->